src/app/item-popup/ItemStat.m.scss

```diff
@@ -83,6 +83,13 @@
   }
 }
 
+.negativeModded {
+  color: #ff3232;
```

Comment:
I *think* we have a red text color defined in `_variables.scss`, but if not we should make one.

Reply:
We do but it is very hard on the eyes. @sundevour brought up that it might be hard for colorblind people so I made this one. TBH I would be all for changing the default one to make it a little lighter. You can see the difference between the first and last screenshots.